.github/ISSUE_TEMPLATE/open-issue.md

```diff
@@ -0,0 +1,8 @@
+---
+name: Open Issue
+about: Open an Issue.
+title: ''
+labels: ''
+assignees: ''
+
+---
```

Comment:
I feel like we could remove this altogether. Arguably if they're not filing a bug, a request or a feature request, maybe it's best they go through discussions.

Reply:
@theo-m done